readthedocs/core/forms.py

```diff
@@ -34,6 +34,18 @@ def __init__(self, *args, **kwargs):
         except AttributeError:
             pass
 
+    def clean_first_name(self):
```

Comment:
You can just use the `max_length` attribute from the `CharField` https://docs.djangoproject.com/en/2.1/ref/forms/fields/#django.forms.CharField

Reply:
Thanks for this wasy way.
I have made the changes.